bg/uni/sofia/fmi/mjt/sentiment/ScoreRepresentation.java

```diff
@@ -0,0 +1,61 @@
+package bg.uni.sofia.fmi.mjt.sentiment;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public enum ScoreRepresentation {
+	UNKNOWN(-1.0, "unknown"), 
+	NEGATIVE(0.0, "negative"), 
+	SNEGATIVE(1.0, "somewhat negative"), 
+	NEUTRAL(2.0, "neutral"),
+	SPOSITIVE(3.0, "somewhat positive"),
+	POSITIVE(4.0, "positive");
+
+	private final double score;
+	private final String scoreDescription;
+
+	private ScoreRepresentation(double score, String scoreDescription) {
+		this.score = score;
+		this.scoreDescription = scoreDescription;
+	}
+
+	private static double roundScore(double reviewScore) {
```

Comment:
Do you really need all that parsing? 
Consider using Math.ceil().
See [https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#ceil-double-] for reference